sklearn/tests/test_multioutput.py

```diff
@@ -85,3 +89,74 @@ def test_multi_target_sample_weights():
 
     X_test = [[1.5,2.5,3.5], [3.5,4.5,5.5]]
     assert_almost_equal(rgr.predict(X_test), rgr_w.predict(X_test))
+
+# Creates a multi-target data set using the iris data set
+# X : The iris predictor data.
+# y : A multi-target(150x3) array generated from the original response data
+
+# Import the data
+iris = datasets.load_iris()
+# create a multiple targets by randomized shuffling and concatenating y.
+X = iris.data
+y1 = iris.target
+y2 = shuffle(y1, random_state=1)
+y3 = shuffle(y1, random_state=2)
+# concatenate the array and transpose
+y = np.vstack((y1, y2, y3)).T
```

Comment:
``np.hstack((y1, y2, y3))`` gives an output of shape ``(450, )`` since it concatenates along axis=0 I changed this to ``np.column_stack((y1, y2, y3))``